src/components/Button/Button.test.tsx

```diff
@@ -5,7 +5,9 @@ import Button from "./Button";
 
 describe("Button", () => {
   it("should display button text", () => {
-    const wrapper = shallow(<Button>Button text</Button>);
+    const wrapper = shallow(
+      <Button onClick={() => jest.fn()}>Button text</Button>
```

Comment:
Can you add a new test that when you click on the button it executes the onclick handler.